entities.lua

```diff
@@ -63,6 +63,9 @@ xp_redo.update_rank_entity = function(player, rank)
 		return
 	end
 
+	local state = player:get_meta():get(xp_redo.HUD_DISPLAY_STATE_NAME)
+	if not (not state or state == "on") then return end
```

Comment:
I think this would be simpler:
```suggestion
	if state == "off" then return end
```

Reply:
thanks, so true :D